testscript/exe.go

```diff
@@ -190,9 +190,14 @@ func (nopTestDeps) WriteProfileTo(name string, w io.Writer, debug int) error {
 func (nopTestDeps) ImportPath() string {
 	return ""
 }
-
 func (nopTestDeps) StartTestLog(w io.Writer) {}
 
 func (nopTestDeps) StopTestLog() error {
 	return nil
 }
+
+// Note: WriteHeapProfile is not needed for Go 1.10.
```

Comment:
shouldn't this rather be "needed to build on Go 1.10, but not actually useful"?

Reply:
yeah, I had the comment wrong. fixed.